t/03-testapi.t

```diff
@@ -608,6 +608,21 @@ subtest 'compat_args' => sub {
     like(warning { testapi::compat_args(\%def_args, [], 'outch') }->[0], qr/^Odd number of arguments/, 'Warned on Odd number 2');
 };
 
+subtest 'check_quiet_option' => sub {
+    $bmwqemu::vars{_QUIET_SCRIPT_CALLS} = 1;
+    my $mock_testapi = Test::MockModule->new('testapi');
+    $mock_testapi->mock(wait_serial => sub {
+            my ($regex, %args) = @_;
+            is($args{quiet}, 1, 'Check default quiet argument');
+            return "XXXfoo\nSCRIPT_FINISHEDXXX-0-";
+    });
+    is(script_output('echo foo', 30), 'foo', '');
+    is(script_run('true'),        '0',   'script_run with no check of success, returns exit code');
+    is(assert_script_run('true'), undef, 'nothing happens on success');
```

Comment:
maybe add `validate_script_output()`, too.

What do you think in testing `_QUIET_SCRIPT_CALLS` is set, but specify `quiet => 0` explicit. I would guess, that then we actually wont `wait_serial()` called with `quiet=>0` as well.

Reply:
> maybe add `validate_script_output()`, too.

sure


> What do you think in testing `_QUIET_SCRIPT_CALLS` is set, but specify `quiet => 0` explicit. I would guess, that then we actually wont `wait_serial()` called with `quiet=>0` as well.
Yes, that is a good point and I would like that the flag overrides the env variable (_QUIET_SCRIPT_CALLS=1 but quiet=>0). I haven't checked that case, but will do it soon.